test/Servant/QuickCheck/InternalSpec.hs

```diff
@@ -135,7 +134,7 @@ type API = ReqBody '[JSON] String :> Post '[JSON] String
 api :: Proxy API
 api = Proxy
 
-type ParamsAPI = QueryParam "one" String :> QueryParam "two" String :> Get '[JSON] String
+type ParamsAPI = QueryParam "one" String :> QueryParam "two" String :> Get '[JSON] ()
```

Comment:
Sorry - I meant the type of the query params could be `()` (so that we don't have to rely on the seed being 0). But I'm okay with this!